Cython/Compiler/Optimize.py

```diff
@@ -2312,6 +2312,38 @@ def _optimise_generic_builtin_method_call(self, node, attr_name, function, arg_l
         return ExprNodes.CachedBuiltinMethodCallNode(
             node, function.obj, attr_name, arg_list)
 
+    PyObject_String_func_type = PyrexTypes.CFuncType(
+        Builtin.str_type, [
+            PyrexTypes.CFuncTypeArg("org", PyrexTypes.py_object_type, None)
```

Comment:
Probably just a typo ("arg"), but let's use "obj" instead (because, why not).
```suggestion
            PyrexTypes.CFuncTypeArg("obj", PyrexTypes.py_object_type, None)
```